contrib/seldon/README.md

```diff
@@ -1,6 +1,14 @@
-# Seldon Kustomize 
+# Seldon Core
 
-## Install Seldon Operator
+[Seldon Core](https://github.com/SeldonIO/seldon-core/) is a framework to deploy your machine learning models on Kubernetes at scale.
+
+# Requirements
+
+* Kubernetes 1.18 - 1.24
+
+Support for Kubernetes 1.25 is currently part of (SeldonIO/seldon-core#4172)[https://github.com/SeldonIO/seldon-core/pull/4172]
```

Comment:
```suggestion
Support for Kubernetes 1.25 is currently part of [SeldonIO/seldon-core#4172](https://github.com/SeldonIO/seldon-core/pull/4172)
```

Reply:
Good catch - thank you for the commit suggest